builtin/logical/pki/storage.go

```diff
@@ -52,6 +57,20 @@ type issuerConfig struct {
 	DefaultIssuerId issuerId `json:"default" structs:"default" mapstructure:"default"`
 }
 
+func genKeyIssuerUUID() (string, error) {
```

Comment:
I'm not clear why this is a separate function - we need to check "err" anyway each time it is called

Reply:
Nevermind, apparently due to needing to panic